components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/internal/APIManagerComponent.java

```diff
@@ -67,10 +67,7 @@
 import org.wso2.carbon.utils.FileUtil;
 
 import javax.cache.Cache;
-import java.io.File;
-import java.io.FilenameFilter;
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```

Comment:
Fixed in #1534(https://github.com/wso2/carbon-apimgt/commit/9db279934bb0f08fa1bd553b4720f2a62474f1ff)